<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>614</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>RePaLk AI</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/logo.png</normaloff>../icons/logo.png</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="toolTipDuration">
   <number>1</number>
  </property>
  <property name="statusTip">
   <string comment="Save to a file"/>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{background-color:qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));
border: 1.5px solid rgb(111, 111, 0);
color:black;}

QDialog:focus{
border: 3px solid rgb(208, 0, 104);
}</string>
  </property>
  <widget class="QPushButton" name="enterButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>545</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="toolTip">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:2px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Press enter to send Prompt Input Widget query and get reponse in Response Output Widget &lt;/p&gt;
&lt;p style=&quot; margin-top:2px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Shortcut key&lt;/span&gt;: &lt;span style=&quot; font-size:10pt; font-weight:600; color:#ff007f;&quot;&gt;Ctrl+return&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(244, 255, 147);
padding:5px;
border: 2px solid rgb(115, 115, 115);
}
QPushButton:hover{
background-color: rgb(208, 0, 104);
}
QPushButton:focus{
border: 3px solid rgb(129, 0, 97);
}

</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/enter.png</normaloff>../icons/enter.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>52</width>
     <height>50</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Ctrl+Return</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>535</y>
     <width>420</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="toolTip">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Prompt Input Widget: &lt;/span&gt;Enter input to get response&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;When Focus On&lt;/span&gt;: Type query and press &lt;span style=&quot; font-size:10pt; font-weight:600; color:#ff007f;&quot;&gt;Ctrl + enter&lt;/span&gt;&lt;span style=&quot; font-weight:600;&quot;&gt; &lt;/span&gt;to send HTTP Request&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;When Focus On: &lt;/span&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Long Press&lt;/span&gt;  &lt;span style=&quot; font-size:10pt; font-weight:600; color:#ff007f;&quot;&gt;Ctrl + Space&lt;/span&gt;&lt;span style=&quot; font-weight:600;&quot;&gt; &lt;/span&gt;for Voice query &lt;span style=&quot; font-weight:600;&quot;&gt;until it say Reconizing Cmd or Screen become blur&lt;/span&gt; and Release to send HTTP Request&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
background-color:rgb(244, 255, 147);
font: italic 12pt &quot;Calibri&quot;;
color:rgb(26, 77, 38);
border-radius:15px;
padding-left:5px;
border: 2px solid rgb(115, 115, 115);
}

QTextEdit:focus{
border: 3px solid rgb(129, 0, 97);
}</string>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
   <property name="lineWrapColumnOrWidth">
    <number>0</number>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Calibri'; font-size:12pt; font-weight:400; font-style:italic;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="placeholderText">
    <string>Enter a Prompt</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>65</y>
     <width>760</width>
     <height>450</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Response Output Widget: &lt;/span&gt;Output HTTP Response for Input request&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Focus On/Off &lt;/span&gt;by Pressing&lt;span style=&quot; font-weight:600;&quot;&gt; &lt;/span&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#ff007f;&quot;&gt;Tab&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;When Focus On: &lt;/span&gt;Press &lt;span style=&quot; font-size:10pt; font-weight:600; color:#ff007f;&quot;&gt;Ctrl + C&lt;/span&gt;&lt;span style=&quot; font-weight:600;&quot;&gt; &lt;/span&gt; to Clear response output widget&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;When Focus On: &lt;/span&gt;Press &lt;span style=&quot; font-weight:600;&quot;&gt;Up or Low arrow Key&lt;/span&gt; to select one response and Press &lt;span style=&quot; font-size:10pt; font-weight:600; color:#ff007f;&quot;&gt;Ctrl + S&lt;/span&gt;&lt;span style=&quot; font-weight:600;&quot;&gt; &lt;/span&gt;to save response in Mentioned formats.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.0568182 rgba(172, 242, 231, 255), stop:0.869318 rgba(255, 215, 215, 255));
padding:4px; 
border: 2px solid rgb(115, 115, 115);
border-radius:10px;
}

QListWidget::item{
padding:4px;
margin:4px;
border-radius: 7px;
 background-color:rgb(244, 180, 26); 
color:rgb(20, 61, 89);
font: 12pt &quot;Arial Rounded MT Bold&quot;;
}
QListWidget:focus{
border: 3px solid rgb(129, 0, 97);
}

QListWidget::item:focus{
border: 3px solid rgb(0,0,255)
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="movement">
    <enum>QListView::Static</enum>
   </property>
   <property name="flow">
    <enum>QListView::TopToBottom</enum>
   </property>
   <property name="isWrapping" stdset="0">
    <bool>false</bool>
   </property>
   <property name="resizeMode">
    <enum>QListView::Fixed</enum>
   </property>
   <property name="layoutMode">
    <enum>QListView::Batched</enum>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>545</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="toolTip">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:2px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Select one output item at a time and click to save into a file&lt;/p&gt;
&lt;p style=&quot; margin-top:2px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Shortcut key&lt;/span&gt;: &lt;span style=&quot; font-size:10pt; font-weight:600; color:#ff007f;&quot;&gt;Ctrl+S&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="statusTip">
    <string extracomment="save to a file"/>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(244, 255, 147);
border: 2px solid rgb(115, 115, 115);
}
QPushButton:hover{
background-color: rgb(208, 0, 104);
}

QPushButton:focus{
border: 3px solid rgb(129, 0, 97);
}

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/Save2file.png</normaloff>../icons/Save2file.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>52</width>
     <height>50</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="voiceButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>545</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="toolTip">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:2px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Press until Voice command show in Output Window&lt;/p&gt;
&lt;p style=&quot; margin-top:2px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Shortcut key&lt;/span&gt;: &lt;span style=&quot; font-size:10pt; font-weight:600; color:#ff007f;&quot;&gt;Ctrl+Space&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(244, 255, 147);
border-radius: 12px;
border: 2px solid rgb(115, 115, 115);
}

QPushButton:hover
{background-color: rgb(208, 0, 104);}

QPushButton:focus{
border: 3px solid rgb(129, 0, 97);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/microphone.png</normaloff>../icons/microphone.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>60</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Ctrl+Space</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>545</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="toolTip">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:2px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Press to clear output window&lt;/p&gt;
&lt;p style=&quot; margin-top:2px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Shortcut key&lt;/span&gt;: &lt;span style=&quot; font-size:10pt; font-weight:600; color:#ff007f;&quot;&gt;Ctrl+C&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(244, 255, 147);
border-radius: 12px;
border: 2px solid rgb(115, 115, 115);
}

QPushButton:hover
{background-color: rgb(208, 0, 104);}

QPushButton:focus{
border: 3px solid rgb(129, 0, 97);
}</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>60</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushBack">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>15</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="toolTip">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:2px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Press to Go back to Main Window&lt;/p&gt;
&lt;p style=&quot; margin-top:2px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Shortcut key: &lt;/span&gt;: &lt;span style=&quot; font-size:10pt; font-weight:600; color:#ff007f;&quot;&gt;Ctlr+B&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(244, 255, 147);
padding:5px;
border: 2px solid rgb(115, 115, 115);
}
QPushButton:hover{
background-color: rgb(208, 0, 104);
}

QPushButton:focus{
border: 3px solid rgb(129, 0, 97);
}
</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/back.png</normaloff>../icons/back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>52</width>
     <height>50</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>700</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>10</pointsize>
     <weight>10</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.0568182 rgba(172, 242, 231, 255), stop:0.869318 rgba(255, 215, 215, 255));
border: 2px solid rgb(115, 115, 115);
border-radius:10px;
font: 87 10pt &quot;Arial Black&quot;;
padding: 4px;
}</string>
   </property>
   <property name="text">
    <string>FineName Upload. Now Chat withyour uploaded Data</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushUpload">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>545</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="toolTip">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:2px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Press to upload file&lt;/p&gt;
&lt;p style=&quot; margin-top:2px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Shortcut key&lt;/span&gt;: &lt;span style=&quot; font-size:10pt; font-weight:600; color:#ff007f;&quot;&gt;Ctlr+U&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(244, 255, 147);
padding:5px;
border: 2px solid rgb(115, 115, 115);
}
QPushButton:hover{
background-color: rgb(208, 0, 104);
}
QPushButton:focus{
border: 3px solid rgb(129, 0, 97);
}

</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/pushData.png</normaloff>../icons/pushData.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>52</width>
     <height>50</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../icons/enter.qrc"/>
 </resources>
 <connections/>
</ui>
